### PR TITLE
Dark Mode

### PR DESCRIPTION
**Pull request by vedant-jain03**
## Related Issuse

- Info about Issue or bug

Closes: #181 

#### Describe the changes you've made

- I have added the dark mode toggle button
- Special feature is that I just invert the color and also it is saving the mode even after refresh the page.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.(Specially for Backend)
- [x] My changes generate no new warnings.

## Screenshots

https://user-images.githubusercontent.com/76901313/115138545-82f43a80-a04a-11eb-8c8d-29ee96c7f7d3.mp4

### CODE REVIEW

**Comment by codewithvk** [comment]
@vedant-jain03 can you re create your PR?

File: client/src/Component/homepage/homepage.js [hunk 1]
```diff
@@ -22,7 +22,17 @@ useEffect(()=>{
...
-       
+    <div className="landing-page-orange">
+
+        <div className="header">
+            <img className="vector-3" src={logo} alt="vector-3" />
+            <div className="home">
+              <img className="vector-4" src={vector5} alt="vector-4" />
+              <div className="place montserrat-bold-white-24px">{place}</div>
+            </div>
+            <div className="Auth">
```
Comment (codewithvk) [inline client/src/Component/homepage/homepage.js:33]: Where is this div over?

File: client/src/Component/homepage/homepage.js [hunk 2]
```diff
@@ -22,7 +22,17 @@ useEffect(()=>{
 
   return (
     <div className={dark ? "landing-page-orange dark-mode":"landing-page-orange"}>
-       
+    <div className="landing-page-orange">
```
Comment (codewithvk) [inline client/src/Component/homepage/homepage.js:25]: Where is this div over?

Comment (codewithvk) [reply client/src/Component/homepage/homepage.js:25]: @vedant-jain03 are you sure in your local your code working properly?

Comment (vedant-jain03) [reply client/src/Component/homepage/homepage.js:25]: Yes It Is working completely fine
I just want to mention that
Code is not clear at all thier is unnecesary code thier which can be converted into clean code

The code is really very messy
Means I have done my part successfully
But if you allow me I can change the code into clear and crystal format and more understandable code

Comment (vedant-jain03) [reply client/src/Component/homepage/homepage.js:25]: Like in code thier are so many divs even only for border thier is div
The navbar section is taking extra div which can be converted into one component

Comment (vedant-jain03) [reply client/src/Component/homepage/homepage.js:25]: Also in css 
For aligning items, left right is used
Which can be converted into one flex box

Comment (codewithvk) [reply client/src/Component/homepage/homepage.js:25]: @vedant-jain03 Ya, You can create a new issue about that.

Comment (vedant-jain03) [reply client/src/Component/homepage/homepage.js:25]: OK I will create a new issue on converting home page js and css into clean code

**Pull request merged**